docs/public-networks/reference/api/index.md

```diff
@@ -4534,7 +4534,7 @@ None
 
 - `target`: _string_ - 32-byte required target boundary condition: 2^256 / difficulty
 
-- `blockNumber`: _string_ - hexadecimal integer representing the current block number
+- `blockNumber`: _string_ - hexadecimal or decimal integer representing the current block number
```

Comment:
```suggestion
- `blockNumber`: _string_ - hexadecimal integer representing the current block number
```

I assume the return values are specifically defined so this description doesn't need an update.